### ISSUE TITLE
on OSX if exuberant ctags is not installed, TagBar will complain

### ISSUE THREAD

**Issue by varunudacity** [jstemmer/gotags#29]
TagBar seems to check the version of ctags before it runs the language specific ctags (go tags), here. As a result, it will complain, that ctags is not "exuberant ctags". If you install ctags, via brew install ctags, then TagBar will proceed to work, and call gotags.

Perhaps this should be filed on tagbar rather than here, however simply noting the issue in the readme.md might be helpful for others. The error message seemed to imply the problem was with gotags, and not ctags itself.

**Comment by jstemmer** [comment]
Yeah, this is really just a Tagbar issue. What was the error message you saw?

**Comment by afshinsa** [comment]
This is the error message that I'm seeing:

Tagbar: Ctags doesn't seem to be Exuberant Ctags!
BSD ctags will NOT WORK. Please download Exuberant Ctags from ctags.sourceforge.net and install it in a directory in your $PATH or set g:tagbar_ctags_bin.
Executed command: "'ctags' --version"
Command output:
/Library/Developer/CommandLineTools/usr/bin/ctags: illegal option -- -
usage: ctags [-BFadtuwvx] [-f tagsfile] file ...
Exit code: 1

**Comment by d0u9** [comment]
Same problem. It seems that `tagbar` still searches `ctags` before invoking `gotags` binary.

**Comment by Casper-Mars** [comment]
Me too. I have set `g:tagbar_ctags_bin` value. And I got error message:

BSD ctags will NOT WORK. Please download Exuberant Ctags from ctags.sourceforge.net and install it in a directory in your $PATH or set g:tagbar_ctags_bin.
Executed command: "'/Users/reachlucifer/go/bin/gotags' --version"
Command output:
flag provided but not defined: -version
gotags version 1.4.1
Usage: /Users/reachlucifer/go/bin/gotags [options] file(s)
  -L string
    ^Isource file names are read from the specified file. If file is "-", input is read from standard in.
  -R^Irecurse into directories in the file list.
  -extra string
    ^Iinclude additional tags with package and receiver name prefixes (+q)
  -f string
    ^Iwrite output to specified file. If file is "-", output is written to standard out.
  -fields string
    ^Iinclude selected extension fields (only +l).
  -list-languages
    ^Ilist supported languages.
  -silent
    ^Ido not produce any output on error.
  -sort
    ^Isort tags. (default true)
  -tag-relative
    ^Ifile paths should be relative to the directory containing the tag file.
  -v^Iprint version.
no file specified
